web/src/components/moreButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function MoreButton({ text, isDisabled, isLeft, isRight }) {
```

Comment:
I don't think you need `isLeft` and `isRight` props.

Pick one and use its presence or absence drive your control flow.